crates/modelardb_server/src/storage/data_transfer.rs

```diff
@@ -489,6 +502,60 @@ mod tests {
         assert_data_transferred(vec![path_1, path_2, path_3], target_dir, data_transfer, 9).await;
     }
 
+    #[tokio::test]
+    async fn test_increase_transfer_batch_size_in_bytes() {
+        let temp_dir = tempfile::tempdir().unwrap();
+        let metadata_manager = create_metadata_manager(temp_dir.path()).await;
+
+        create_compressed_file(metadata_manager.clone(), temp_dir.path()).await;
+        create_compressed_file(metadata_manager.clone(), temp_dir.path()).await;
+
+        let (_, mut data_transfer) =
+            create_data_transfer_component(metadata_manager, temp_dir.path()).await;
+
+        data_transfer
+            .set_transfer_batch_size_in_bytes(COMPRESSED_FILE_SIZE * 10)
+            .await
+            .unwrap();
+
+        assert_eq!(
+            data_transfer.transfer_batch_size_in_bytes,
+            COMPRESSED_FILE_SIZE * 10
+        );
+
+        // Data should not be transferred.
```

Comment:
Maybe "Data should not have been transferred."

Reply:
Fixed.